app/controllers/api/v1/reference/regions_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```

Comment:
What is the reasoning behind the `reference` namespace as opposed to directly using `regions` etc., as I can see we are using it for `locations` as well?

Reply:
It's just a convention - all of the reference data endpoints live in this module.

Reply:
spoke in person: we use `reference` for resources where data is static and won't be amended by any API calls